code/player/TTTPlayer.cs

```diff
@@ -5,26 +5,41 @@
 
 namespace TTTGamemode
 {
-    public partial class Player : BasePlayer
+	public enum Role { None, Innocent, Detective, Traitor }
+	
+    public partial class TTTPlayer : Player
     {
-        public enum Role { None, Innocent, Detective, Traitor }
-
-        public Body Body { get; set; }
-        public Role Role { get; set; } = Player.Role.None;
+	    public Body Body { get; set; }
+        public Role CurrentRole { get; set; }
         public int Credits { get; set; } = 0;
 
         private TimeSince _timeSinceDropped;
         private DamageInfo _lastDamageInfo;
 
-        public Player()
+        public TTTPlayer()
         {
-            Inventory = new Inventory(this);
-            Animator = new StandardPlayerAnimator();
+	        Inventory = new Inventory(this);
 
-            Role = Role.None;
```

Comment:
Yea you renamed it to avoid issues with the `Role` enum (something I documented in the Trello as well), but I think `Role` would be more intuitive. Instead, we probably should move or rename the `Role` enum.